runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/stores/RemoteFileStore.java

```diff
@@ -21,8 +21,9 @@
 import org.apache.reef.tang.annotations.DefaultImplementation;
 
 /**
- * Interface for remote block stores (e.g., GlusterFS, ...).
+ * Interface for remote block stores (e.g., GlusterFS, CrailFS...).
  */
+
 @DefaultImplementation(GlusterFileStore.class)
```

Comment:
Actually `RemoteFileStore` is constructed by default, and using Crail as the default would cause every job to fail when there is no CrailFileSystem detected on the nodes that run the job. So maybe `GlusterFileStore` can be the default?